pkgs/tools/misc/vdirsyncer/default.nix

```diff
@@ -3,13 +3,13 @@
 # Packaging documentation at:
 # https://github.com/untitaker/vdirsyncer/blob/master/docs/packaging.rst
 pythonPackages.buildPythonApplication rec {
-  version = "0.10.0";
+  version = "0.11.0";
   name = "vdirsyncer-${version}";
   namePrefix = "";
```

Comment:
could you remove this as we're using `buildPythonApplication` which already removes the prefix.